File: snippets/visualbasic/VS_Snippets_CLR_System/system.decimal.operators/vb/decrement2.vb
﻿' Visual Basic .NET Document
Option Strict On

' <Snippet6>
Module Example
   Public Sub Main()
      Dim number As Decimal = 1079.8d
      Console.WriteLine("Original value:    {0:N}", number)
      Console.WriteLine("Decremented value: {0:N}", Decimal.Subtract(number, 1))
   End Sub
End Module
' The example displays the following output:
'       Original value:    1,079.80
'       Decremented value: 1,078.80
' </Snippet6>

File: snippets/fsharp/System/Decimal/op_Addition/decrement2.fs
module decrement2

// <Snippet6>
open System

let number = 1079.8m
printfn $"Original value:    {number:N}"
printfn $"Decremented value: {Decimal.Subtract(number, 1):N}"

// The example displays the following output:
//       Original value:    1,079.80
//       Decremented value: 1,078.80
// </Snippet6> 